.gitignore

```diff
@@ -3,3 +3,4 @@
 *.DS_Store
 npm-debug.log
 node_modules
+dist
```

Comment:
The dist directory should be deleted once deployment is complete, so this shouldn't be needed.